scripts/BUILD.bazel

```diff
@@ -42,3 +43,19 @@ kt_jvm_binary(
         "//scripts/src/java/org/oppia/android/scripts/ci:compute_affected_tests_lib",
     ],
 )
+
+REGEX_PATTERN_CHECK_ASSETS = retrieve_regex_pattern_check_assets("regex_pattern_assets")
+
+kt_jvm_library(
+    name = "regex_check_assets",
+    data = REGEX_PATTERN_CHECK_ASSETS,
+    visibility = [":oppia_script_binary_visibility"],
+)
+
+kt_jvm_binary(
+    name = "regex_pattern_validation_check",
+    testonly = True,
+    data = REGEX_PATTERN_CHECK_ASSETS,
```

Comment:
Could you add ``:regex_check_assets`` as a runtime dep instead of passing the assets along in ``data`` again? Ideally we should avoid passing in the assets twice.

Reply:
@BenHenning 
Hm. On doing this, for some reason it is not able to find the generated `.pb` files at the run time. When we run the `kt_jvm_binary`. It is giving this error when we add `regex_check_assets` as a runtime dep.
```
Exception in thread "main" java.io.FileNotFoundException: scripts/assets/filename_pattern_validation_checks.pb (No such file or directory)
        at java.base/java.io.FileInputStream.open0(Native Method)
        at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
        at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
        at org.oppia.android.scripts.regex.RegexPatternValidationCheckKt.loadProto(RegexPatternValidationCheck.kt:98)
        at org.oppia.android.scripts.regex.RegexPatternValidationCheckKt.retrieveFilenameChecks(RegexPatternValidationCheck.kt:65)
        at org.oppia.android.scripts.regex.RegexPatternValidationCheckKt.main(RegexPatternValidationCheck.kt:35)
```

Reply:
Probably a limitation in our custom rule. It seems like this won't be an option for now, then.